libs/related-products/state/testing/src/mock-related-products.facade.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+import { DaffProduct } from '@daffodil/product';
+import { DaffRelatedProductsFacadeInterface } from '@daffodil/related-products/state';
+
+/**
+ * @inheritdoc
+ */
+@Injectable({ providedIn: 'root' })
+export class MockDaffRelatedProductsFacade implements DaffRelatedProductsFacadeInterface {
```

Comment:
I think this needs an extra comment to explain what the mock facade is for.

Reply:
changed